draft-ietf-jsonpath-base.md

```diff
@@ -156,8 +156,9 @@ their UTF-8 encoding.
 For example, the Unicode PLACE OF INTEREST SIGN (U+2318) would be defined
 in ABNF as `%x2318`.
 
-
-The terminology of {{-json}} applies.
+The terminology of {{-json}} applies, including definitions for
```

Comment:
Please indicate where 8259 defines "string".

Reply:
https://datatracker.ietf.org/doc/html/rfc8259#section-7

Reply:
Yes.

```
   A string begins and ends with
   quotation marks.  All Unicode characters may be placed within the
   quotation marks, except for the characters that MUST be escaped:
   quotation mark, reverse solidus, and the control characters (U+0000
   through U+001F).
```

These are not the strings JSONPath uses.

Reply:
I think the ones you're thinking of relate to "names" in a bracket-selector or filter expression.  These are currently discussed in the "union selector" -> "child" -> "syntax" section, where it's given effectively the same definition as 8259 except that single quotes are also allowed.

I see the argument to add a "string" definition to account for the support of single-quotes.